Jshaughn/severity

jshaughn opened the pull request:
HWKALERTS-53 :  Add Severity to Alerting

lucasponce commented on hawkular-alerts-engine/src/main/java/org/hawkular/alerts/engine/impl/CassAlertsServiceImpl.java:
```diff
@@ -205,6 +209,10 @@ private void initPreparedStatements() throws Exception {
         }
+        if (insertAlertSeverity == null) {
+            insertAlertSeverity = session.prepare("INSERT INTO " + keyspace + ".alerts_severities " +
+                    "(alertId, severity) VALUES (?, ?) ");
```
Missing insert tenantId.
TenantId is used as partition key, to distribute data across nodes. Data needs to have affinity, so with tenantId we can have all data related to a tenant on a same node, avoiding that a query should be spread across multiple nodes and executing a map/reduce to get result.

jshaughn commented:
Lucas, I've fixed the query issues you pointed out.  Also, I've now merged this PR on master (the refactor with CassStatement), and added a bit more alerts query testing.  I think this PR is ready to merge, please perform any further review you deem necessary and merge if you see fit.
